src/client/components/AlvisGraph.tsx

```diff
@@ -308,7 +310,7 @@ export class AlvisGraph extends React.Component<
 
       this.props.onAgentClick(internalId);
 
-      event.consume(); // TODO: what it does?
+      // event.consume(); // TODO: what it does?
```

Comment:
This is not part of the fix - I'm not sure if we shoude consume this event, and it works without consumsion thus I commented it out for now.